spinta/backends/postgresql/commands/write.py

```diff
@@ -22,7 +22,11 @@ async def insert(
     table = backend.get_table(model)
     async for data in dstream:
         patch = commands.before_write(context, model, backend, data=data)
-
+        if 'coordinates' in patch.keys():
```

Comment:
What if a property is not named `coordinates`? See my testing notes, I named it `point`.